src/Components/test/E2ETest.Playwright/Microsoft.AspNetCore.Components.E2ETests.Playwright.csproj

```diff
@@ -0,0 +1,72 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(RepoRoot)src\Testing\src\build\Microsoft.AspNetCore.Testing.props" />
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>disable</Nullable>
+    <TestDependsOnAspnetPackages>true</TestDependsOnAspnetPackages>
+    <TestDependsOnAspnetRuntime>true</TestDependsOnAspnetRuntime>
+    <TestDependsOnAspnetAppPackages>false</TestDependsOnAspnetAppPackages>
+    <ContainsFunctionalTestAssets>false</ContainsFunctionalTestAssets>
+    <IsTestProject>true</IsTestProject>
+    <TestGroupName>Components.E2ETests.Playwright</TestGroupName>
+    <DefineConstants>$(DefineConstants); PLAYWRIGHT_ADAPTER</DefineConstants>
+
+    <TestTrimmedApps Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</TestTrimmedApps>
+    <TestTrimmedApps Condition="'$(Configuration)' == 'Release'">true</TestTrimmedApps>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Hosting" />
+    <Reference Include="Microsoft.AspNetCore.Server.Kestrel" />
+    <Reference Include="Microsoft.AspNetCore.StaticFiles" />
+    <ProjectReference Include="..\..\WebAssembly\testassets\StandaloneApp\StandaloneApp.csproj" />
+    <ProjectReference Include="$(RepoRoot)src\Shared\BrowserTesting\src\Microsoft.AspNetCore.BrowserTesting.csproj" />
+    <ProjectReference Include="..\testassets\TestServer\Components.TestServer.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\E2ETest\Infrastructure\ServerFixtures\*.cs" LinkBase="Infrastructure\ServerFixtures" />
+    <Compile Include="..\E2ETest\Tests\StandaloneAppTest.cs" LinkBase="Tests" />
```

Comment:
Here you can see we're actually using the real `StandaloneAppTest.cs` test fixture, and it now does run and succeed on Playwright.